ui/src/components/sermons/schedules/EasterSchedule.js

```diff
@@ -0,0 +1,64 @@
+import {
```

Comment:
can you move this whole folder inside easter folder

Reply:
done